src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,151 @@
 package core.basesyntax;
 
+import core.basesyntax.model.User;
+import core.basesyntax.service.ImpossibleRegisterUserException;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorldTest {
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+    private User user = new User();
+
+    @BeforeEach
+    void setUp() {
+        user.setId(12l);
+        user.setLogin("sanya67");
+        user.setAge(18);
+        user.setPassword("123456");
+    }
+
+    @Test
+    void loginOfUserAlreadyExists_NotOk() {
```

Comment:
The test method name `loginOfUserAlreadyExists_NotOk` does not follow the required naming convention `<methodUnderTest>_<state>_<expectedBehavior>`. Consider renaming it to something like `register_LoginAlreadyExists_ThrowsException`.